hc/api/tests/test_admin.py

```diff
@@ -7,16 +7,23 @@ class ApiAdminTestCase(BaseTestCase):
     def setUp(self):
         super(ApiAdminTestCase, self).setUp()
         self.check = Check.objects.create(user=self.alice, tags="foo bar")
+        self.alice.is_staff = True
+        self.alice.is_superuser = True
+        self.alice.save()
 
-        ### Set Alice to be staff and superuser and save her :)
+        # Set Alice to be staff and superuser and save her :)
 
     def test_it_shows_channel_list_with_pushbullet(self):
         self.client.login(username="alice@example.org", password="password")
 
         Channel.objects.create(user=self.alice, kind="pushbullet",
                                value="test-token")
+        self.assertTrue("pushbullet",
```

Comment:
For this test you should be checking if `pushbullet` is listed as a channel kind."pushbullet" is specified as the kind for the channel created on [line 19](https://github.com/andela/healthchecks-boomslang/pull/14/files#diff-2f22a4dcc02da39a417848dce63db42cR19).
Please take note how `assertTrue` works => `assertTrue(expr, msg=None)`. The function only checks whether the first argument evaluates to true. The second argument is the message that should be displayed.